crates/pathfinder/src/consensus/inner/p2p_task.rs

```diff
@@ -578,10 +579,21 @@ struct DeferredExecution {
 }
 
 /// Proposal commitment and the address of its proposer.
-#[derive(Debug, Copy, Clone, Default)]
+#[derive(Debug, Clone)]
 struct ProposalCommitmentWithOrigin {
     pub proposal_commitment: ProposalCommitment,
     pub proposer_address: ContractAddress,
+    pub pol_round: Round,
+}
+
+impl Default for ProposalCommitmentWithOrigin {
+    fn default() -> Self {
+        Self {
+            proposal_commitment: ProposalCommitment::default(),
+            proposer_address: ContractAddress::default(),
+            pol_round: Round::nil(),
```

Comment:
wonder if `Round` shouldn't have `Default` as well...

Reply:
> What would that mean? `0` or `nil`?
 
`nil`, since that's how it's used...

> Imho, I'd rather we're explicit about it.

...but I don't have a strong preference...